src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -26,6 +27,27 @@
       :on-press            #(rf/dispatch [:navigate-to :add-address-to-watch])
       :add-divider?        true}]]])
 
+(defn share-qr-code
```

Comment:
hmm, I'm not sure why we have this code here. 
Also the naming `share-qr-code` is more implementation details then what it is actually doing. Share qr code relates to how you get to this view but really this is a bottom sheet for scanned-qr-actions.